src/artery/envmod/GlobalEnvironmentModel.h

```diff
@@ -7,6 +7,8 @@
 #ifndef GLOBALENVIRONMENTMODEL_H_
 #define GLOBALENVIRONMENTMODEL_H_
 
+
+// #include "artery/envmod/sensor/Sensor.h"
```

Comment:
I guess this line was accidentally added.

Reply:
Whoops